python/ffsim/operators/fermi_hubbard.py

```diff
@@ -0,0 +1,81 @@
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fermi-Hubbard model Hamiltonian."""
+
+from ffsim._lib import FermionOperator
+from ffsim.operators.fermion_action import cre_a, cre_b, des_a, des_b
+
+
+def fermi_hubbard(
```

Comment:
It just occurred to me that this is only the one-dimensional Fermi-Hubbard model. I might need to rethink the API, because we definitely want to support the 2d model at some point. Let's rename this function to `fermi_hubbard_1d` for now.